patterns/solid/src/main/java/com/baeldung/l/advanced/Account.java

```diff
@@ -0,0 +1,10 @@
+package com.baeldung.l.advanced;
```

Comment:
Package name of `l` is probably not great. How about `liskov`?

Reply:
OK - this is a slightly tricky one.
These packages names were introduced for the examples used in https://www.baeldung.com/solid-principles article e.g. _com.baeldung.s, com.baeldung.o, com.baeldung.l_ and so on. 
I put my examples in a new package under_com.baeldung.l_ called _com.baeldung.l.advanced_. 

As much as I did not like the original package names, I did not want to include package name refactoring at least in the first pass of this PR. Now that you have reviewed the code for this article, I can do the refactoring as a separate commit. Please let me know what you think.